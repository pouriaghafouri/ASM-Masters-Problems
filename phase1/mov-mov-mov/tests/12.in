3453621322605639524 -8128195827310128839
