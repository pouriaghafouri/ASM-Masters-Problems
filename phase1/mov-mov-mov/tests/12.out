-6864926923793783253
